### PR TITLE
Side padding: Added missing percentage

### PR DESCRIPTION
**Pull request by RyuuPendragon**
Adding 5% side padding, as 10% side padding making webtoons thin to read.

### CODE REVIEW

File: app/src/main/java/eu/kanade/tachiyomi/ui/setting/SettingsReaderController.kt [hunk 1]
```diff
@@ -319,12 +319,13 @@ class SettingsReaderController : SettingsController() {
                 titleRes = R.string.pref_webtoon_side_padding
                 entriesRes = arrayOf(
                     R.string.webtoon_side_padding_0,
+                    R.string.webtoon_side_padding_5,
```
Comment (Jays2Kings) [inline app/src/main/java/eu/kanade/tachiyomi/ui/setting/SettingsReaderController.kt:322]: this actually needs to be added in the strings xml too

**Review by Jays2Kings — CHANGES_REQUESTED** [review]

File: app/src/main/java/eu/kanade/tachiyomi/ui/setting/SettingsReaderController.kt (hunk 1 above)
Comment (RyuuPendragon) [reply app/src/main/java/eu/kanade/tachiyomi/ui/setting/SettingsReaderController.kt:322]: @Jays2Kings Do I need to manually add the missing stings in all languages

Comment (Jays2Kings) [reply app/src/main/java/eu/kanade/tachiyomi/ui/setting/SettingsReaderController.kt:322]: Just the main strings xml

Comment (RyuuPendragon) [reply app/src/main/java/eu/kanade/tachiyomi/ui/setting/SettingsReaderController.kt:322]: Added the missing percentage string to mail strings.xml

**Pull request merged**